phase4-entsog-client/src/etc/license-template.txt

```diff
@@ -0,0 +1,14 @@
+Copyright (C) 2020-2021 Philip Helger (www.helger.com)
+philip[at]helger[dot]com
```

Comment:
You should add your name here. Than call `mvn licence:format` to get it in all files